terraform/modules/compute/wls_compute/variables.tf

```diff
@@ -16,6 +16,11 @@ variable "compartment_id" {
   description = "The OCID of the compartment where the compute will be created"
 }
 
+variable "create_policies" {
+  type        = bool
+  description = "Set to true to create OCI IAM policies and dynamic groups required by the WebLogic for OCI stack. If this is set to false, the policies and dynamic groups need to be created manually"
```

Comment:
I think you should set a default value here. I think true is the default in the UI so you should set to true as the default.
My concern is for the CLI case. The absence of a value is represented in Terraform by the keyword null so if someone doesn't specify this variable I'm not sure how the code that retrieves it would behave. I guess it would treat a null like a false since the code I saw in orahub does "create_polices == 'true'" test.

Reply:
"create_policies" variable is first defined in terraform/variables.tf file with the default value set to true, always a boolean value will be passed to module "compute" from terraform/main.tf.  So, we don't need to handle that case in terraform/modules/compute/wls_compute/variables.tf.

Reply:
Got it. I should have noticed that this wasn't the main variables.tf file.